integration/patch_test.go

```diff
@@ -189,6 +197,12 @@ func patch(t *testing.T, ref, patchedTag, path string, ignoreErrors bool, report
 		reportPath = "-r=" + path + "/scan.json"
 	}
 
+	if strings.Contains(ref, "oracle") && reportFile {
+		err := errors.New("oracle is not supported with trivy scans")
+		fmt.Println(err)
```

Comment:
instead of this, lets test "Error: Detected Oracle image passed in" since copa fast fails anyways, this will help with codecov